src/Core/Form/IdentifiableObject/DataProvider/CombinationFormDataProvider.php

```diff
@@ -123,7 +124,7 @@ private function extractStockData(CombinationForEditing $combinationForEditing,
             'options' => [
                 'stock_location' => $stockInformation->getLocation(),
                 'low_stock_threshold' => $stockInformation->getLowStockThreshold(),
-                'low_stock_alert' => $stockInformation->isLowStockAlertEnabled(),
+                sprintf('%slow_stock_threshold', DisablingSwitchExtension::FIELD_PREFIX) => $stockInformation->isLowStockAlertEnabled(),
```

Comment:
This one shouldn't be needed The disabling switch is supposed to be initialized based on the value of the associated field So it shouldn't be split into two data, simply indicating that `'low_stock_threshold' => 0,` should automatically disable it